modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/toLocaleDateTimeFormateed.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+export function toLocalDateTimeFormatted(
+	dateUTC: string,
+	language: string,
+	userTimezone: string
+) {
+	const date = new Date(dateUTC);
+
+	const options = {
+		day: '2-digit',
+		hour: '2-digit',
+		minute: '2-digit',
+		month: 'short',
+		timeZone: userTimezone,
+		year: 'numeric',
+	} as const;
+
+	const timeZone = new Intl.DateTimeFormat(language, options);
+
+	return timeZone.format(date);
+}
```

Comment:
the name of the file should be `toLocaleDateTimeFormatted.ts` instead of `toLocaleDateTimeFormateed.ts`

Reply:
Hey rebeca, I renamed it `toLocalDateTimeFormatted.ts` without the 'e' from toLocal**e**.

Which is the name of the file that contains this function in the application. I believe that `LocalDateTime` is a better fit than `LocaleDateTime`, but I would like to hear your opinion.

Reply:
indeed it is a better name, you can leave as `toLocalDateTimeFormatted.ts`